src/components/helpers.tsx

```diff
@@ -18,7 +18,11 @@ export const block = (margin: number = 0) => {
 
 const sizes = theme.flexboxgrid.breakpoints
 
-export const media: any = Object.keys(sizes).reduce((accumulator, label) => {
+type Media = {
+  [S in keyof typeof sizes]: typeof css
+}
```

Comment:
that `keyof typeof` trick is very useful to use object keys as an union type.
thanks for the detailed explanation